app/scripts/lib/experiment.js

```diff
@@ -13,11 +13,15 @@ define(function (require, exports, module) {
   const FORCE_EXPERIMENT_GROUP_PARAM = 'forceExperimentGroup';
   const UA_OVERRIDE = 'FxATester';
 
-  const ALL_EXPERIMENTS = {
+  const STARTUP_EXPERIMENTS = {
```

Comment:
Document `STARTUP_EXPERIMENTS` and when it's used. Update the document for `chooseExperimements` with some of that knowledge too.